src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -138,9 +140,25 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
   const interactiveId = embeddable.ref_id;
 
   let proposedHeight: number;
-  const nativeHeight = embeddableData?.native_height || 0;
-  const nativeWidth = embeddableData?.native_width || 0;
-  const aspectRatio = nativeHeight && nativeWidth ? nativeWidth / nativeHeight : kDefaultAspectRatio;
+  let nativeHeight: number;
+  let nativeWidth: number;
+  let aspectRatio: number;
+
+  if (embeddable.type === "ManagedInteractive") {
+    nativeHeight = embeddable.custom_native_height || 0;
```

Comment:
@lublagg 

Sorry, I meant that custom_native_height and custom_native_width need to check their inherit booleans.

So the logic should be like:
`nativeHeight = (embeddable.inherit_native_height ? embeddable.library_interactive?.data.native_height : embeddable.custom_native_height) || 0;`

Or you could use the fact that `embeddableData = embeddable.library_interactive?.data` in this case. So then it can be:
`nativeHeight = (embeddable.inherit_native_height ? embeddableData?.native_height : embeddable.custom_native_height) || 0;`

And then apply this same change to native_width.